docs/user/dashboard/lens.asciidoc

```diff
@@ -303,3 +303,22 @@ Pagination in a data table is unsupported in *Lens*. However, the <<types-of-vis
 ===== Is it possible to have more than one y-axis scale in visualizations?
 
 *Lens* lets you pick, for each Y dimension, up to two distinct axis: *left* and *right*. Each axis can have a different scale.
+
+[float]
+[[why-my-value-is-not-of-the-right-color-using-dynamic-coloring]]
+===== Why mt value is not of the right color when using dynamic coloring?
```

Comment:
Are we using the term "dynamic coloring" anywhere else? I don't think it's self-explanatory

Reply:
Fair point. I assumed here it will be in the release notes.
Probably I'd suggest then `datatable coloring`, wdyt?

Reply:
Sounds good to me. Other things that would work IMHO:
* conditional coloring
* value-based coloring
* table cell coloring